### PR TITLE
Improve documentation of e3.vcs

### PR DESCRIPTION
**Pull request by Nikokrock**

### CODE REVIEW

File: e3/vcs/svn.py [hunk 1]
```diff
@@ -68,11 +68,16 @@ def local_url(self, repo_path):
     def create(cls, repo_path, initial_content_path=None):
         """Create a local subversion repository.
 
-        :param repo_path: a local directory that contains the repository
+        This create a local repository (not a working copy) that can be
```
Comment (brobecke) [inline e3/vcs/svn.py]: create -> creates

File: e3/vcs/svn.py [hunk 2]
```diff
@@ -68,11 +68,16 @@ def local_url(self, repo_path):
...
+        referenced by using file:// protocol. The purpose of the this function
+        is mainly to test svn related function without relying on a remote
```
Comment (brobecke) [inline e3/vcs/svn.py]: "svn related" -> "svn-related"
"function" -> "functions"

File: e3/vcs/svn.py (hunk 1 above)
Comment (Nikokrock) [reply e3/vcs/svn.py]: Fixed

File: e3/vcs/svn.py (hunk 2 above)
Comment (Nikokrock) [reply e3/vcs/svn.py]: Fixed

**Pull request merged**